iModelCore/ECDb/Tests/NonPublished/CommonTableExpTests.cpp

```diff
@@ -792,4 +792,33 @@ TEST_F(CommonTableExpTestFixture, alias_to_cte) {
     }
 }
 
+//---------------------------------------------------------------------------------------
+// @bsiclass
+//+---------------+---------------+---------------+---------------+---------------+------
+TEST_F(CommonTableExpTestFixture, SubQueryBlock) {

```

Comment:
Hm the test covers the problem scenario but seems rather small.
I wonder if there are other variations of this that we should cover. Like, more complex nesting, or a simple comparison instead of IN (?).